entrypoint.sh

```diff
@@ -10,18 +10,26 @@ if [[ -z "${PROXY_LIST_URL}" ]]; then
     echo "Using mounted proxy list"
     touch $BASE_DIR/proxy-list.txt
     echo " ---> Done"
-else
+elif [[ -z "${BRIGHT_DATA_API_TOKEN}"]]; then
```

Comment:
This reads a bit weird, why not keep this as the else?

Reply:
Could be.. I didn't see a big difference

Reply:
Changed